Octokit.Tests/Clients/IssuesClientTests.cs

```diff
@@ -60,13 +71,29 @@ public void SendsAppropriateParameters()
                         && d["filter"] == "assigned"
                         && d["sort"] == "created"
                         && d["state"] == "open"
-                        && d["direction"] == "asc"));
+                        && d["direction"] == "asc"),
+                    Args.ApiOptions);
             }
         }
 
         public class TheGetAllForOwnedAndMemberRepositoriesMethod
         {
             [Fact]
+            public async Task EnsuresNonNullArguments()
+            {
+                var client = new IssuesClient(Substitute.For<IApiConnection>());
+
+                await Assert.ThrowsAsync<ArgumentNullException>(
+                    () => client.GetAllForOwnedAndMemberRepositories((ApiOptions)null));
+                await Assert.ThrowsAsync<ArgumentNullException>(
+                    () => client.GetAllForOwnedAndMemberRepositories((IssueRequest)null));
+                await Assert.ThrowsAsync<ArgumentNullException>(
+                    () => client.GetAllForOwnedAndMemberRepositories(null, new ApiOptions()));
+                await Assert.ThrowsAsync<ArgumentNullException>(
+                    () => client.GetAllForOwnedAndMemberRepositories(new IssueRequest(), null));
```

Comment:
really minor nitpick but the equivalent code in other unit tests in this file are all on one line, while these ones are indented/split...